portal/src/texts/core/pattern.md

```diff
@@ -19,3 +19,29 @@ Måten vi bruker, fjerner eller legger til data i et datasett, kan vi også stan
 ### Overflytsregler
 
 Når en tekst er for lang for et bestemt område, kan vi bestemme hvordan vi vil at dette skal vises. Vi kan for eksempel forkorte informasjonen og vise en ellipse. I Jøkul finner du blant annet mønstre for hvordan vi forkorter søkebaner og lange URL-er.
+
+
+# Skjemadesign
+
+Skjema utgjør en viktig del av Fremtinds løsninger. For å sikre konsistens på tvers har vi laget noen regler for skjemaoppsett bruk av skjemakomponentene i Jøkul.
```

Comment:
Reconsider the last sentence.

`skjemaoppsettbruk av skjemakomponentene`

thats rather heavy

Reply:
Suggest we just delete "skjemaoppsett"

Reply:
Sett gjerne opp tekster dere skaper til språkvask hos meg, så tar jeg en runde på det når jeg er hos dere. Denne kan jeg nok få gjort noe med :-)